packages/sources/peregrine-fund-admin/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { Adapter } from '@chainlink/external-adapter-framework/adapter'
+import { config } from './config'
+import { nav, reserve } from './endpoint'
+
+export const adapter = new Adapter({
+  //Requests will direct to this endpoint if the `endpoint` input parameter is not specified.
+  defaultEndpoint: nav.name,
+  // Adapter name
+  name: 'PEREGRINE_FUND-ADMIN',
+  // Adapter configuration (environment variables)
+  config,
+  // List of supported endpoints
+  endpoints: [nav, reserve],
```

Comment:
Need to add a rate limiter here, otherwise your API is going to get hammered

for example


```
  rateLimiting: {
    tiers: {
      default: {
        rateLimit1m: 30,
      },
    },
  },
```

Reply:
Thank you, added.